js/bitmex.js

```diff
@@ -228,6 +244,137 @@ module.exports = class bitmex extends Exchange {
+        //            "asset": "XBT",
+        //            "currency": "XBt",
+        //            "majorCurrency": "XBT",
+        //            "name": "Bitcoin",
+        //            "currencyType": "Crypto",
+        //            "scale": "8",
+        //            // "mediumPrecision": "8",
+        //            // "shorterPrecision": "4",
+        //            // "symbol": "₿",
+        //            // "weight": "1",
+        //            // "tickLog": "0",
+        //            "enabled": true,
+        //            "isMarginCurrency": true,
+        //            "minDepositAmount": "10000",
+        //            "minWithdrawalAmount": "1000",
+        //            "maxWithdrawalAmount": "100000000000000",
+        //            "networks": [
+        //                {
+        //                    "asset": "btc",
+        //                    "tokenAddress": "",
+        //                    "depositEnabled": true,
+        //                    "withdrawalEnabled": true,
+        //                    "withdrawalFee": "20000",
+        //                    "minFee": "20000",
+        //                    "maxFee": "10000000"
+        //                }
+        //            ]
+        //        },
+        //     }
+        //
+        const result = {};
+        const keys = Object.keys (response);
+        for (let i = 0; i < keys.length; i++) {
+            const key = keys[i];
+            const currency = response[key];
+            const id = this.safeString (currency, 'asset');
+            const code = this.safeCurrencyCode (id);
+            const name = this.safeString (currency, 'name');
+            const chains = this.safeValue (currency, 'networks', []);
+            let depositEnabled = false;
+            let withdrawEnabled = false;
+            const networks = {};
+            const scale = this.safeString (currency, 'scale');
+            const precisionString = this.parsePrecision (scale);
+            const precision = this.parseNumber (this.parsePrecision (scale));
+            for (let j = 0; j < chains.length; j++) {
+                const chain = chains[j];
+                const networkId = this.safeString (chain, 'asset');
+                const network = this.networkIdToCode (networkId);
+                const withdrawalFeeRaw = this.safeString (chain, 'withdrawalFee');
+                const withdrawalFee = this.parseNumber (Precise.stringMul (withdrawalFeeRaw, precisionString));
+                const isDepositEnabled = this.safeValue (chain, 'depositEnabled', false);
+                const isWithdrawEnabled = this.safeValue (chain, 'withdrawalEnabled', false);
+                const active = (isDepositEnabled && isWithdrawEnabled);
+                if (isDepositEnabled) {
+                    depositEnabled = true;
+                }
+                if (isWithdrawEnabled) {
+                    withdrawEnabled = true;
+                }
+                networks[network] = {
```

Comment:
@ttodua the unified network code is not used here ⬇️ 
![image](https://user-images.githubusercontent.com/43336371/199523501-ea96f66a-0fc1-4454-a56a-77dfdda60e7d.png)

Reply:
hm, how it comes that I have different result? 
[!img](https://imgsh.net/a/X4xtEiq.png)

Reply:
@ttodua can you double check this pls? I still get the same result
![image](https://user-images.githubusercontent.com/43336371/207835444-f5f76161-bedf-47ef-a6eb-47e90aec5e8c.png)


I think this is due to the fact `networksById` does not contain these networks yet

Reply:
@carlosmiei weird, what should I do? i see unified codes: https://imgsh.net/a/Nhjf17d.png

Reply:
@ttodua are you sure that you have the same exact same version locally? your repo might be ahead with some un-pushed changes